scripts/coredump_gen_handler.py

```diff
@@ -110,13 +110,22 @@ def parse_ts_dump_name(self, ts_stdout):
         syslog.syslog(syslog.LOG_ERR, "stdout of the 'show techsupport' cmd doesn't have the dump name")
         return ""
 
-    def invoke_ts_cmd(self, since_cfg):
-        since_cfg = "'" + since_cfg + "'"
+    def invoke_ts_cmd(self, since_cfg, num_retry=0):
         cmd_opts = ["show", "techsupport", "--silent", "--since", since_cfg]
         cmd  = " ".join(cmd_opts)
         rc, stdout, stderr = subprocess_exec(cmd_opts, env=ENV_VAR)
-        if rc:
+        if rc == EXT_LOCKFAIL:
+            syslog.syslog(syslog.LOG_NOTICE, "Another instance of techsupport running, aborting this. stderr: {}".format(stderr))
+            return ""
+        elif rc == EXT_RETRY:
+            if num_retry <= MAX_RETRY_LIMIT:
+                print(num_retry)
```

Comment:
Should there be a gap of few seconds before next retry?

Reply:
Not required, EXT_RETRY happening is less likely and response should be quick in order to grab the lock. so i don't think a gap is required. 
I'll remove the print statement though